packages/fast-components/src/slider-label/slider-label.ts

```diff
@@ -0,0 +1,58 @@
+import { attr, FastElement, observable } from "@microsoft/fast-element";
+import { FASTSlider } from "../slider";
+import { SliderOrientation } from "../slider/slider";
+
+export class SliderLabel extends FastElement {
+    @observable
+    public positionStyle: string;
+
+    @observable
+    public root: HTMLDivElement;
```

Comment:
I don't think you need to observe refs (@eisenbergEffect to keep me honest)

Reply:
No need to observe refs, since the template engine is just setting that property, rather than needing to update based on when it changes.

Reply:
done, it does not need to be there as suggested.